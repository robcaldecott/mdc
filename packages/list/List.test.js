import React from "react";
import {
  List,
  ListItem,
  ListItemText,
  ListItemSecondaryText,
  ListItemGraphic,
  ListItemMeta,
  ListDivider
} from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <List twoLine dense className="custom-list">
        <ListItem activated>
          <ListItemGraphic>
            <i className="material-icons">inbox</i>
          </ListItemGraphic>
          <ListItemText>
            Text
            <ListItemSecondaryText>Line 2</ListItemSecondaryText>
          </ListItemText>
        </ListItem>
        <ListDivider inset />
        <ListItem selected>Text</ListItem>
        <ListDivider />
        <ListItem className="custom-list-item">
          Text
          <ListItemMeta>
            <i className="material-icons">delete</i>
          </ListItemMeta>
        </ListItem>
      </List>
    )
  ).toMatchSnapshot();
});
