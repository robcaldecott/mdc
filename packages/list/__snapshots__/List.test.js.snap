// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<ul
  className="mdc-list mdc-list--dense mdc-list--two-line custom-list"
>
  <li
    className="mdc-list-item mdc-list-item--activated"
  >
    <i
      className="mdc-list-item__graphic material-icons"
    >
      inbox
    </i>
    <span
      className="mdc-list-item__text"
    >
      Text
      <span
        className="mdc-list-item__secondary-text"
      >
        Line 2
      </span>
    </span>
  </li>
  <li
    className="mdc-list-divider mdc-list-divider--inset"
    role="separator"
  />
  <li
    className="mdc-list-item mdc-list-item--selected"
  >
    Text
  </li>
  <li
    className="mdc-list-divider"
    role="separator"
  />
  <li
    className="mdc-list-item custom-list-item"
  >
    Text
    <i
      className="mdc-list-item__meta material-icons"
    >
      delete
    </i>
  </li>
</ul>
`;
